react/core/main.tsx

```diff
@@ -115,14 +115,14 @@ const render = async (
     page,
     pages,
     extensions,
-    culture: { locale },
   } = runtime
 
   const cacheControl = canUseDOM ? undefined : new PageCacheControl()
   const baseURI = getBaseURI(runtime)
   registerEmitter(runtime, baseURI)
   emitter = runtime.emitter
-  addLocaleData(locale)
+
+  await import('../intl-polyfill')
```

Comment:
We should only do it if browser does not implement `Intl.PluralRules` or `Intl.RelativeTimeFormat`. Otherwise we are spending network bandwidth with an asset that is useless.

It should also be done out of here. We don't need to wait for the render function to be called. We want these assets to be fetched as soon as we discover the browser needs a polyfill.

Reply:
Also, check if this is running on client-side before doing it.